packages/broker/test/unit/plugins/operator/OperatorFleetState.test.ts

```diff
@@ -0,0 +1,122 @@
+import { OperatorFleetState } from '../../../../src/plugins/operator/OperatorFleetState'
+import { mock, MockProxy } from 'jest-mock-extended'
+import StreamrClient, { MessageListener, Subscription } from 'streamr-client'
+import { toEthereumAddress, wait, waitForCondition, waitForEvent } from '@streamr/utils'
+import { toStreamID } from '@streamr/protocol'
+import { eventsWithArgsToArray } from '@streamr/test-utils'
+
+const ADDRESS = toEthereumAddress('0x61BBf708Fb7bB1D4dA10D1958C88A170988d3d1F')
```

Comment:
Could we use `randomEthereumAddress()` here, or do we need this address specifically?